### PR TITLE
Removing Outdated Demo and Adding New Demo

### PR DESCRIPTION
**Pull request by adrianmahjour**
Removing Demo Code for an old demo that is outdated and should not be used

### CODE REVIEW

**Comment by adrianmahjour** [comment]
Added new demo content for new WKC & AutoAI Demo

File: In_Db2_Machine_Learning/Automated AI Model Development with IBM Cloud Pak for Data and Db2/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                  "PAYMENTMETHOD" VARCHAR(50 OCTETS) ,
+                  "MONTHLYCHARGES" DOUBLE ,
+                  "TOTALCHARGES" DOUBLE ,
+                  "CHURN" VARCHAR(6 OCTETS) );"
+                
+db2 "IMPORT FROM TelcoCustomers.csv OF DEL skipcount 1 INSERT INTO <table.schema>.<table_name>"
+```
+
+## Reference Video
+[Automated AI Model Development with IBM Cloud Pak for Data and Db2](LINK)
```
Comment (kdrodger) [inline In_Db2_Machine_Learning/Automated AI Model Development with IBM Cloud Pak for Data and Db2/README.md:47]: Missing link?

Comment (adrianmahjour) [reply In_Db2_Machine_Learning/Automated AI Model Development with IBM Cloud Pak for Data and Db2/README.md:47]: @kdrodger yes, the link will be updated once the demo video is published. Hopefully by end of next week

Comment (kdrodger) [reply In_Db2_Machine_Learning/Automated AI Model Development with IBM Cloud Pak for Data and Db2/README.md:47]: No problem with that, and not a deal breaker but a broken/non-existent link isn't a great experience.  For the time being I'll suggest either just removing that line entirely, or at least put some text there indicating that the link is still to come.  Thanks

Comment (adrianmahjour) [reply In_Db2_Machine_Learning/Automated AI Model Development with IBM Cloud Pak for Data and Db2/README.md]: Understood. I've removed the link for now

**Review by kdrodger — APPROVED** [review]

**Pull request merged**